docs/source/training.rst

```diff
@@ -279,7 +279,8 @@ Finally, you can view the results, including any calculated metrics, by launchin
 ``logging_dir`` directory.
 
 Trainer Integrations
```

Comment:
I didn't read properly last time and didn't catch this to `training.rst` instead of `main_classes/trainer.rst`. `training.rst` is a tutorial focused on fine-tuning a model on a given dataset, so it doesn't make sense to have all this content here. Could you move it to `main_classes/trainer.rst`?